app/(afterLogin)/(employeeInformation)/employees/manage-employees/[id]/_components/job/index.tsx

```
@@ -172,7 +174,15 @@ function Job({ id }: { id: string }) {
         />
       </Card>
       <WorkScheduleComponent id={id} />
-      <CreateEmployeeJobInformation id={id} />
+      <CreateEmployeeJobInformation 
```

ESLint

[error] Delete `·`

(prettier/prettier)